src/dmd/mtype.d

```diff
@@ -231,7 +231,6 @@ private enum TFlags
     real_        = 8,
     imaginary    = 0x10,
     complex      = 0x20,
-    char_        = 0x40,
```

Comment:
Doesn't this need removing from headers too?

Reply:
I don't think it's in the headers anywhere?  Grepping for char_ and Tflags in the headers doesn't seem to show anything

Reply:
It won't have the same name, search for the enum instead.

Reply:
Typebasic has an unsigned flags field, which I assume corresponds to this enum, but I don't think the enum + its values are defined anywhere in mtype.h

Reply:
It wasn't added to mtype.h as part of the previous PR: https://github.com/dlang/dmd/pull/9889/files#diff-4cc7c37241d3613798dafeb87aa3014a

Reply:
Ok then.